contracts/ERC677BridgeToken.sol

```diff
@@ -57,21 +57,28 @@ contract ERC677BridgeToken is
     function transfer(address _to, uint256 _value) public returns (bool)
     {
         require(superTransfer(_to, _value));
-        if (isContract(_to) && !contractFallback(_to, _value, new bytes(0))) {
+        callAfterTransfer(msg.sender, _to, _value);
+        return true;
+    }
+
+    function transferFrom(address _from, address _to, uint256 _value) public returns (bool) {
```

Comment:
Since we modify the core functionality of the contract consider to increase the version returned by `getTokenInterfacesVersion`.

Reply:
updated version https://github.com/poanetwork/poa-bridge-contracts/pull/220/commits/94c3e02364ea570d953054341e17707aef5230bd